lua/completion/hover.lua

```diff
@@ -297,16 +297,16 @@ local function callback_function(_, method, result)
       return bufnr, winnr
     end)
   else
-    M.default_callback(_, method, result, _)
+    M.default_handler(_, method, result, _)
   end
 end
 
 M.autoOpenHoverInPopup = function()
   if vim.fn.pumvisible() ~= 1 then return end
   for _, client in pairs(vim.lsp.buf_get_clients(0)) do
-    local default_callback = client.config.callbacks['textDocument/hover'] or vim.lsp.callbacks['textDocument/hover']
-    if default_callback ~= callback_function then
-      client.config.callbacks['textDocument/hover'] = callback_function
```

Comment:
So I'm not super familiar with neovim lsp and completion but when I inspect the client locally the callbacks are on the client table directly (e.g. `client.callbacks`) and not nested under `client.config.callbacks`. I'm not sure where handlers came from but my nightly installation doesn't name callbacks handlers. I'm using asdf to install neovim, though so very probably I'm doing something different.